Algorithm.CSharp/CustomVolatilityModelAlgorithm.cs

```diff
@@ -91,17 +91,22 @@ public void Update(Security security, BaseData data)
             {
                 _needsUpdate = false;
                 var mean = _window.Average();
-                var std = Math.Sqrt((double)_window.Sum(x => (x - mean)*(x - mean)) / _window.Count()); 
+                var std = Math.Sqrt((double)_window.Sum(x => (x - mean)*(x - mean)) / _window.Count());
                 Volatility = Convert.ToDecimal(std * Math.Sqrt(252d));
```

Comment:
I suggest using `SafeDecimalCast` to convert `double` to `decimal`:
```csharp
Volatility = (std * Math.Sqrt(252d)).SafeDecimalCast();
```
to avoid `OverflowException`.